.sass-lint.yml

```
@@ -2,7 +2,7 @@ files:
   include:
     - 'src/legacy/core_plugins/metrics/**/*.s+(a|c)ss'
     - 'src/legacy/core_plugins/timelion/**/*.s+(a|c)ss'
-    - 'src/legacy/ui/public/vislib/**/*.s+(a|c)ss'
+    - 'src/legacy/core_plugins/kbn_vislib_vis_types/public/vislib/**/*.s+(a|c)ss'
```

I think this file path can just be simplified to 
```suggestion
    - 'src/legacy/core_plugins/kbn_vislib_vis_types/**/*.s+(a|c)ss'
```

---

I still want files in `src/legacy/core_plugins/kbn_vislib_vis_types/` to be linted. In this case, I moved the `vislib` library which was ignored for whatever reason, but it is going to be phased out and deleted eventually.

---

Yes those files will still be linted, my suggestion would just start looking at a higher level folder (all the way down to vislib) in case other SASS files are added outside of vislib.

---

Right but wouldn't we want new scss files added to the plugin to be linted?

If I do...

```yml
- 'src/legacy/core_plugins/kbn_vislib_vis_types/**/*.s+(a|c)ss'
```

No s{a|c}ss files in `kbn_vislib_vis_types` would be linted. Right? Is that what we want?

---

This section is for **inclusion** of files not **exclusion**. Therefore, any files within `kbn_vislib_vis_types/**` **will** be linted.

---

We are slowly turning SASS linting on because there's a lot of files that have problems. It was too hard to do a whole repo turn on then exclude files. That's why we've went the inclusion route instead.

---

Wooow thanks sorry I missed that it was for inclusion. Yup you are right, thanks for the clarification.